src/youtube/js/youtube.js

```diff
@@ -86,6 +86,7 @@ require("../../../html5-common/js/utils/constants.js");
    * @property {object} youtubePlayer The youtube player is creted.
    */
   function onYouTubeIframeAPIReady() {
+    if ( youtubeID == null || youtubePlayer ) { return};
```

Comment:
Ah, just noticed the double equal null. It might be good to add a comment saying "We expect youtubeID to be initialized" or something similar. Also, should there be a console log in addition to returning early? Might help debugging.